.github/workflows/master.yml

```diff
@@ -57,31 +82,66 @@ jobs:
         key: ${{ matrix.java }}-${{ matrix.hadoop }}-maven-io-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ matrix.java }}-${{ matrix.hadoop }}-maven-io-
+    - uses: actions/cache@v1
+      with:
+        path: ~/.ivy2/cache
+        key: ${{ matrix.java }}-${{ matrix.hadoop }}-ivy2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ matrix.java }}-${{ matrix.hadoop }}-ivy2-
     - name: Set up JDK ${{ matrix.java }}
       uses: actions/setup-java@v1
       with:
         java-version: ${{ matrix.java }}
-    - name: Build with Maven
+    # PySpark
+    - uses: actions/setup-python@v2
+      if: contains(matrix.modules, 'pyspark')
```

Comment:
Will this branch be reached? The module names are "pyspark-..."

Reply:
This one:

https://github.com/HyukjinKwon/spark/pull/4/files/7f4a4c0b87445204343c901b7c3d6dd55e2fac34#diff-04eb107ee163a50b61281ca08f4e4c7bR41-R43

It checks if the string contains `pyspark`